aqua/beekeeper-studio/Portfile

```diff
@@ -32,13 +33,27 @@ checksums           ${distname}${extract.suffix} \
                     sha256  093a41aa23e2d4b2259265a0a9fbe3feef8cfd1e4054417f39bedc0f768d5c1b \
                     size    42107147
 
-depends_build       port:yarn
+depends_build       port:go \
+                    port:yarn
 
 use_configure       no
 
 build {
+    set gopath ${workpath}/go
+
+    # Set up all JS dependencies
     system -W ${worksrcpath} "yarn"
-    system -W ${worksrcpath} "CSC_IDENTITY_AUTO_DISCOVERY=false yarn run electron:build --mac dir"
+
+    # Workaround for: https://trac.macports.org/ticket/60555
+    # Build app-builder-bin locally and insert it into node_modules
+    system "GOPATH=${gopath} go get -v github.com/develar/app-builder"
+    file delete ${worksrcpath}/node_modules/app-builder-bin/mac/app-builder
+    file link ${worksrcpath}/node_modules/app-builder-bin/mac/app-builder \
+              ${gopath}/bin/app-builder
+
+    # Build electron app
+    system -W ${worksrcpath} \
+        "CSC_IDENTITY_AUTO_DISCOVERY=false yarn run electron:build --mac dir"
```

Comment:
Why not add `CSC_IDENTITY_AUTO_DISCOVERY=false` to the `build.env`?

Reply:
It doesn't seem as if `build.env` environment variables apply to commands run using `system`.  I just moved `GOPATH` and `CSC_IDENTITY_AUTO_DISCOVERY` to `build.env`  and that confirmed that issue.  Not sure if this has always been the case, or if this is a regression.

Reply:
I've also seen `build.env` not always work as you might expect, and had to resort to what you do here. So if it works I'm fine with leaving as is.